main.go

```diff
@@ -156,6 +157,11 @@ func main() {
 		glog.Fatalf("Error: %s", err)
 	}
 
+	if *versionFlag {
+		fmt.Printf("kube-state-metrics version: %#v", version.GetVersion())
```

Comment:
`kube-state-metrics version: %#v` -> `%#v`

I prefer we do not add any prefix and keep it simple.